eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -39,8 +39,8 @@ case class RemoteChanges(proposed: List[UpdateMessage], acked: List[UpdateMessag
   def all: List[UpdateMessage] = proposed ++ signed ++ acked
 }
 case class Changes(ourChanges: LocalChanges, theirChanges: RemoteChanges)
-case class HtlcTxAndSigs(txinfo: HtlcTx, localSig: ByteVector64, remoteSig: ByteVector64)
-case class PublishableTxs(commitTx: CommitTx, htlcTxsAndSigs: List[HtlcTxAndSigs])
+case class HtlcTxAndSigs(txinfo: HtlcTx, remoteSig: ByteVector64)
+case class PublishableTxs(commitTx: CommitTx, remoteSig: ByteVector64, htlcTxsAndSigs: List[HtlcTxAndSigs])
```

Comment:
Should we rename the commitTx field to make it very explicit that it's unsigned (and prevent someone changing it in the future)?
```scala
case class PublishableTxs(unsignedCommitTx: CommitTx, remoteSig: ByteVector64, htlcTxsAndSigs: List[HtlcTxAndSigs])
```

Reply:
I did that with 644e3f8, the goal was to minimize the changeset for easier review.

Reply:
:+1: I like `require` approach, it's better than a rename